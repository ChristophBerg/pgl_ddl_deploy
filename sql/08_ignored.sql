SET ROLE test_pgl_ddl_deploy;

CREATE TEMP TABLE foo(id SERIAL PRIMARY KEY);
ALTER TABLE foo ADD COLUMN bla TEXT;
DROP TABLE foo;

SELECT set_name, ddl_sql_raw, ddl_sql_sent FROM pgl_ddl_deploy.events ORDER BY id DESC LIMIT 10;
