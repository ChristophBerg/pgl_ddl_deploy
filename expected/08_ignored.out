SET ROLE test_pgl_ddl_deploy;
CREATE TEMP TABLE foo(id SERIAL PRIMARY KEY);
ALTER TABLE foo ADD COLUMN bla TEXT;
DROP TABLE foo;
SELECT 1 AS myfield INTO TEMP foo;
DROP TABLE foo;
CREATE TEMP TABLE foo AS
SELECT 1 AS myfield;
DROP TABLE foo;
SELECT set_name, ddl_sql_raw, ddl_sql_sent FROM pgl_ddl_deploy.events ORDER BY id DESC LIMIT 10;
 set_name |          ddl_sql_raw           |          ddl_sql_sent          
----------+--------------------------------+--------------------------------
 test4    | DROP TABLE foo CASCADE;        | DROP TABLE foo CASCADE;
 test3    | DROP TABLE foo CASCADE;        | DROP TABLE foo CASCADE;
 test2    | DROP TABLE foo CASCADE;        | DROP TABLE foo CASCADE;
 test1    | DROP TABLE foo CASCADE;        | DROP TABLE foo CASCADE;
 test8    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
 test7    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
 test6    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
 test5    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
 test4    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
 test3    | DROP TABLE foobar.foo CASCADE; | DROP TABLE foobar.foo CASCADE;
(10 rows)

